scripts/generate-supplier-user-csv.py

```diff
@@ -32,10 +32,10 @@
 sys.path.insert(0, '.')
 
 from dmscripts.helpers.auth_helpers import get_auth_token
-from dmscripts.bulk_upload_documents import get_bucket_name
 from dmscripts.generate_supplier_user_csv import generate_csv_and_upload_to_s3
 from dmscripts.helpers import logging_helpers
 from dmscripts.helpers.logging_helpers import logging
+from helpers.s3_helpers import get_bucket_name
```

Comment:
Should this be dmscripts.helpers?
```suggestion
from dmscripts.helpers.s3_helpers import get_bucket_name
```

Reply:
That's weird, Pycharm autosuggested it with no problems. Will fix